lib/workers/repository/update/branch/index.ts

```diff
@@ -366,6 +377,11 @@ export async function processBranch(
         prNo: branchPr?.number,
         result: BranchResult.NoWork,
       };
+    } else if (config.recreateMergedPr) {
+      logger.debug(
+        'Rebase branch cause the commits from old branch have already been merged.'
+      );
+      config.reuseExistingBranch = false;
```

Comment:
Is this for the scenario where the new PR is *identical* to the old one, *and* the old branch hasn't been deleted?

Reply:
Yes